### PR TITLE
Add static variable for res.users model

### PR DESCRIPTION
**Pull request by harunyasar**
@joel-depiltech I moved the hardcoded user model name to a static variable. In addition I think have to replace all snake case methods name to camel case.

### CODE REVIEW

File: src/OdooClient.php [hunk 1]
```diff
@@ -28,6 +28,12 @@ class OdooClient
     private static $_execute = 'execute_kw';
 
     /**
+     * res.user model
+     * @var string $_user_model
+     */
+    private static $_user_model = 'res.users';
```
Comment (joel-depiltech) [inline src/OdooClient.php]: Good idea !

File: src/OdooClient.php [hunk 2]
```diff
@@ -213,7 +219,7 @@ public function getUid()
      * Message creator for XML-RPC request
      * @return xmlrpcmsg Message header
      */
-    private function _execute()
+    private function createMessageHeader()
```
Comment (joel-depiltech) [inline src/OdooClient.php]: I really prefer this name. It's very smart and explicit !

**Comment by joel-depiltech** [comment]
Nice work !
I'm not sure about rename all snake case method to camel case, because it's very easy to match with method and call to web service. Maybe just transform in camel case the methods which do not match web service call.

**Comment by harunyasar** [comment]
@joel-depiltech you're right. just private methods can be camel case.

**Pull request merged**